src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final String DEF_FIGURE_COLOR = Color.WHITE.name().toLowerCase();
```

Comment:
Consider using a more descriptive name for 'DEF_FIGURE_COLOR'. The name should reflect that it is the default color for figures.